apps/frontend/src/app/3_organisms/ZeroLocForm/CloseCreditLine.tsx

```diff
@@ -157,6 +161,14 @@ export const CloseCreditLine: FC<CloseCreditLineProps> = ({
         />
       )}
 
+      {isRecoveryMode && (
```

Comment:
Adjust the condition on line `153` to `hasError && !isRecoveryMode` because in case both `hasError` and `isRecoveryMode` are true, you will see 2 `ErrorBadge` and the first one is not needed.